scripts/pre_commit_linter.py

```diff
@@ -1218,6 +1216,61 @@ def _check_directive_scope(all_files):
     return summary_messages
 
 
+def _match_line_breaks_in_controller_dependencies(all_files):
+    """This function checks whether the line breaks between the dependencies
+    listed in the controller of a directive or service exactly match those
+    between the arguments of the controller function.
+    """
+    print 'Starting controller dependency check'
+    print '----------------------------------------'
+    files_to_check = [
+        filename for filename in all_files if not
+        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
+        and filename.endswith('.js')]
+    failed = False
+    summary_messages = []
+
+    # For RegExp explanation, please see https://regex101.com/r/T85GWZ/2/.
+    pattern_to_match = (
+        r'controller: \[(?P<stringfied_dependencies>[\S\s]*?)' +
+        r'function\((?P<function_parameters>[\S\s]*?)\)')
+    for filename in files_to_check:
+        with open(filename) as f:
+            content = f.read()
+        matched_patterns = re.findall(pattern_to_match, content)
+        for matched_pattern in matched_patterns:
+            stringfied_dependencies, function_parameters = matched_pattern
+            stringfied_dependencies = (
+                stringfied_dependencies.strip().replace(
+                    "'", '').replace(' ', ''))[:-1]
+            function_parameters = function_parameters.strip().replace(' ', '')
+            if stringfied_dependencies != function_parameters:
+                failed = True
+                print (
+                    '%s --> Please ensure that line breaks between '
+                    'the stringfied dependencies,"%s" and the function '
```

Comment:
use a colon instead of a comma after `dependencies` so that the dependencies are more visible.

Reply:
Done, thanks!